ui/nuxeo-pagination-controls.js

```diff
@@ -42,18 +44,56 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
             display: block;
           }
 
+          .container {
+            @apply --layout-horizontal;
+            @apply --layout-center;
+          }
+
           .controls {
             color: var(--nuxeo-text-default, #000);
             font-size: 1rem;
-            padding: 0 4px;
+            @apply --layout-horizontal;
+            @apply --layout-center;
           }
 
-          paper-icon-button {
-            padding: 8px;
+          nuxeo-select {
+            width: 3rem;
+
+            --paper-dropdown-menu-input: {
+              padding: 0;
+              width: 2rem;
+            }
+
+            --paper-input-container: {
+              padding: 0;
+            }
+
+            --paper-input-container-shared-input-style: {
+              max-width: 2rem;
+            }
+
+            --paper-input-container-underline: {
+              display: none;
+            }
+
+            --paper-input-container-underline-focus: {
+              display: none;
+            }
+
+            --paper-input-container-shared-input-style: {
+              font-size: 1rem;
+              vertical-align: bottom;
+              margin-top: 1px !important;
```

Comment:
I know important is not a good pattern but I couldn't find another way to add the margin-top to the element (or another way to align the numbers)

Reply:
you need to override the mixin set by `paper-input`, something along these lines:
```suggestion
              margin: 1px 0 0 0;
```
https://github.com/PolymerElements/paper-input/blob/master/paper-input-container.js#L29

Reply:
indeed! that works! thanks @semisse !